docs/getting-started.md

```diff
@@ -41,24 +41,37 @@ Antrea deployment manifest:
 
 ## Theia Installation
 
+In Theia v0.1.0, we introduce the Flow Aggregator Helm Chart to Theia
+From Theia v0.2.0 and Antrea v1.8.0, the Flow Aggregator Helm Chart is moved from Theia to Antrea.
+
 To enable both [Grafana Flow Collector](network-flow-visibility.md) and
-[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please install
-Flow Aggregator and Theia by runnning the following commands:
+[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please follow
+the steps below:
 
-```bash
-git clone https://github.com/antrea-io/theia.git
-helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
-helm install theia theia/build/charts/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
-```
+1. Download the Flow Aggregator Helm Chart and the Theia Helm Chart.
 
-To enable only Grafana Flow Collector, please install Flow Aggregator and Theia
-by runnning the following commands:
+    For Theia v0.1.0, please clone the repository and checkout branch `release-0.1`.
+    The Helm charts are located under the folder `build/charts`.
 
-```bash
-git clone https://github.com/antrea-io/theia.git
-helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true -n flow-aggregator --create-namespace
-helm install theia theia/build/charts/theia -n flow-visibility --create-namespace
-```
+    From Theia v0.2.0 and Antrea v1.8.0, the Flow Aggregator Helm Chart is moved from
+    Theia repository to Antrea repository; and the Helm Charts are added to release
+    assets. Please download `theia-chart.tgz` from [Theia release assets](https://github.com/antrea-io/theia/releases)
```

Comment:
I wonder if letting users download the helm charts is inconvenient.
When using other helm charts, they usually provide a `helm repo add` command first and then a `helm install` command.
May I ask what steps we need to have similar process?

Reply:
The way to turn our GitHub project into a self-hosted Helm chart repo I found is to use [helm/chart-releaser-action](https://github.com/helm/chart-releaser-action). But it is a little bit different in the release process. Instead of generating a chart release when we create the release from a branch, it automatically checks each push and releases the chart when the chart version number is changed. So currently I use the way Antrea repo uses to create the release, which simply packages the charts and uploads it.

Reply:
Starting with Antrea v1.8.0, users will be able to do `helm repo add charts.antrea.io`. This is in progress. See https://github.com/antrea-io/website/pull/25.

Reply:
Got it, thanks Yanjun and Antonin.

Reply:
Have not noticed that. Thanks Antonin for the information!

Reply:
Hi @antoninbas I took a look at the PR quickly, it seems only the antrea helm chart is added. Could we also have the flow-aggregator chart from Antrea repo and theia chart from Theia repo added to the website?

Reply:
Yes, but we are testing the mechanism end to end first. We can add the 2 remaining charts afterwards.

Reply:
Got it. Thanks!